### DISCUSSION TITLE
Compatibility to bbb 2.6

### DISCUSSION THREAD

**Discussion by chandraes** [blindsidenetworks/scalelite#898] [Q&A]
Are this scalelite loadbalancer still compatible with bbb 2.6?

**Comment by jfederico** [comment]
It is. But it all depends of how you deploy. There is a bug in 1.4 that will be solved in 1.4.2 related to the recording formats supportd by the image scalelite-nginx.

**Comment by JeanPluzo** [comment]
Hi,

I just upgraded my BBB servers from 2.5 to 2.6. Because of the bug @jfederico mentions, I also upgraded SL to 1.4.2. Until now everything is working correctly. Just my 2cts.

Regards,
J.
